federatedscope/core/configs/cfg_data.py

```diff
@@ -10,15 +10,18 @@ def extend_data_cfg(cfg):
 
     cfg.data.root = 'data'
     cfg.data.type = 'toy'
+    cfg.data.args = []  # args for external dataset, eg. [{'download': True}]
+    cfg.data.splitter = ''
+    cfg.data.splitter_args = []  # args for splitter, eg. [{'alpha': 0.5}]
+    cfg.data.transform = []  # transform for x, eg. [['ToTensor'], ['Normalize', {'mean': [0.1307], 'std': [0.3081]}]]
+    cfg.data.target_transform = []  # target_transform for y, use as above
+    cfg.data.pre_transform = []  # pre_transform for `torch_geometric` dataset, use as above
```

Comment:
for the three args, how about naming them as ``feat_transform``, ``label_transform``, and ``graph_pre_transform``?

Reply:
I can't agree. For the `torchvision` and `torch_geometric` dataset, we will use `**transform_funcs` as their args directly.